### PR TITLE
DP-9176 [a11y] Focus on accordion open/close button

### PR DESCRIPTION
**Pull request by ygannett**
Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)

## Description

Add the same treatment for the contact list accordion button (@molecules/contact-us.twig) to the binder page table of content accordion button for screen readers to avoid reading the symbol in the button and give screen reader users more useful text information for the button function.

Add **aria-label** attribute  to `span.show` and `span.hide` in `<button`> to override the symbols.
Remove `span.ma__visually-hidden` which is replaced with the `aria-label`.

No visual change. 
No Drupal change involved.

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-9176)

## Steps to Test

1. Go to http://localhost:3000/?p=pages-binder-page-internal.
2. Check the markup for the table of content button.
3. Find the aria-label values as (screenshots are available below):
     - "click to show table of contents" with "+"
     - "click to hide table of contents" with "x" 

## Screenshots
Show
show

Hide
hide

## Additional Notes:

Anything else to add?

#### Impacted Areas in Application

* 

#### @TODO

*

### CODE REVIEW

File: patternlab/styleguide/source/_patterns/03-organisms/by-template/table-of-contents-overlay.twig [hunk 1]
```diff
@@ -4,10 +4,7 @@
     {% set decorativeLink = tableOfContentsOverlay.title %}
     {% include "@atoms/decorative-link.twig" %}
     <button type="button" class="ma__toc__toc__toggle js-inline-overlay-toggle" aria-expanded="false" aria-controls="toc-overlay">
-      <span class="show">
-        +
-        <span class="ma__visually-hidden">Show table of contents</span>
-      </span>
+      <span class="show" aria-label="click to show table of contents">+</span>
```
Comment (mrossi113) [inline patternlab/styleguide/source/_patterns/03-organisms/by-template/table-of-contents-overlay.twig:7]: Capitalize the first letter in the aria-label sentence.

File: patternlab/styleguide/source/_patterns/03-organisms/by-template/table-of-contents-overlay.twig [hunk 2]
```diff
@@ -16,9 +13,8 @@
       {% set decorativeLink = tableOfContentsOverlay.title %}
       {% include "@atoms/decorative-link.twig" %}
       <button type="button" class="ma__toc__toc__toggle js-inline-overlay-toggle" aria-expanded="true" aria-controls="toc-overlay">
-        <span class="hide">
+        <span class="hide" aria-label="click to hide table of contents">
```
Comment (mrossi113) [inline patternlab/styleguide/source/_patterns/03-organisms/by-template/table-of-contents-overlay.twig:16]: Capitalize the first letter in the aria-label sentence.

**Review by mrossi113 — CHANGES_REQUESTED** [review]
The first letter in both of the aria-label are lower case. Could you capitalize the word `click` in both?

File: patternlab/styleguide/source/_patterns/03-organisms/by-template/table-of-contents-overlay.twig (hunk 1 above)
Comment (ygannett) [reply patternlab/styleguide/source/_patterns/03-organisms/by-template/table-of-contents-overlay.twig:7]: It's for read-aloud only.  Capitalization doesn't matter.

**Review by mrossi113 — APPROVED** [review]

**Comment by mrossi113** [comment]
@ygannett sorry I lost track of this PR!  I'm going to S&M this in develop for next Mayflower release.

**Pull request merged**